GeneralsMD/Code/GameEngine/Source/GameNetwork/GameSpy/Thread/BuddyThread.cpp

```diff
@@ -504,6 +504,7 @@ void BuddyThreadClass::errorCallback( GPConnection *con, GPErrorArg *arg )
 static void getNickForMessage( GPConnection *con, GPGetInfoResponseArg *arg, void *param )
 {
 	BuddyResponse *resp = (BuddyResponse *)param;
+	static_assert(ARRAY_SIZE(arg->nick) <= ARRAY_SIZE(resp->arg.message.nick), "Incorrect array size");
```

Comment:
Would be nice to invert their order to match strcpy argument order. This make it easier to read.

Multiple times in this change.